e2e/select_test.go

```diff
@@ -15,6 +15,16 @@ import (
 	"gotest.tools/v3/icmd"
 )
 
+type testcase struct {
+	name          string
+	cmd           []string
+	informat      string
+	structure     string
+	compression   bool
+	outformat     string
+	expectedValue string
+}
+
```

Comment:
Each test leverages it's own testcase type. Moving this type definition in order to reuse in the `TestSelectWithEmptyBucket` is not feasible, since `TestSelectWithEmptyBucket` does not require `informat`, `structure`, `compression`, `outformat` or `expectedValue`, it only asserts that the stdout should be empty and process exists with 0. Thus, moving this struct back to where it belongs and creating a new struct in the `TestSelectWithEmptyBucket` that only has the required fields is better, also it's easier to read the tests later on.

There is a [good article](https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html) if you want to look at on reusing components on tests 😄